heartbeat.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/moira-alert/cache/filter"
+)
+
+func heartbeat(db *filter.DbConnector, terminate chan bool, wg *sync.WaitGroup) {
+	defer wg.Done()
+	count := filter.TotalMetricsReceived.Count()
+	for {
+		select {
+		case <-terminate:
+			return
+		case <-time.After(time.Second * 5):
```

Comment:
Why do we need to check heartbeat that often? In case of low load metrics can arrive once every minute.